dll/shellext/zipfldr/CZipExtract.cpp

```diff
@@ -390,9 +399,16 @@ class CZipExtract :
             CStringA FullPath = CombinedPath;
             FullPath.Replace('/', '\\');    /* SHPathPrepareForWriteA does not handle '/' */
             DWORD dwFlags = SHPPFW_DIRCREATE | (is_dir ? SHPPFW_NONE : SHPPFW_IGNOREFILENAME);
+        RetryPrepare:
             HRESULT hr = SHPathPrepareForWriteA(hDlg, NULL, FullPath, dwFlags);
             if (FAILED_UNEXPECTEDLY(hr))
             {
+                char StrippedPath[MAX_PATH] = { 0 };
```

Comment:
```suggestion
                CHAR StrippedPath[MAX_PATH] = { 0 };
```

Reply:
I'd leave this type as is, according to existing code style, like in variable defined above:
```c
char CombinedPath[MAX_PATH * 2] = { 0 };
```

Reply:
Rest is addressed :)